Moonshot/player/LedgeWallDetector.gd

```diff
@@ -23,20 +25,32 @@ func _unhandled_input(event: InputEvent) -> void:
 
 
```

Comment:
Maybe for consistency with these methods call the ray cast methods in the order they are attached to the player:

```
ray_top
ray_middle
ray_bottom
```

Just incase we have to debug this code again then ordering of the methods is one less thing to keep track of

Reply:
makes sense